### PR TITLE
explicitly list package data

### PR DESCRIPTION
**Pull request by bryan-brancotte**
when ran locally with setuptools 69.0.3 html file are now included (not working with 48.0.0)

### CODE REVIEW

**Comment by bryan-brancotte** [comment]
Hi @smithdc1 

Following #7 I updated the package so templates are included.

Although work might be needed to actually [publish a new version](https://github.com/django-crispy-forms/crispy-bootstrap3/blob/main/.github/workflows/publish.yml#L48-L66). Do you think its doable ?

Best whishes

**Comment by smithdc1** [comment]
Hi @bryan-brancotte  -- I'm happy to build the package locally and push it to PyPI.

Probably at that point we could then archive this repo? Bootstrap 2 was EOL some time ago. https://endoflife.date/bootstrap

🤷

**Comment by bryan-brancotte** [comment]
Hi @smithdc1 

Sorry, late reply. 

I fully agree with you, considering the long passed EOL of bootstrap2, we could build the package one last time properly and then mark it as archived :up: 

Best,

File: setup.py [hunk 1]
```diff
@@ -18,6 +18,12 @@
         "Framework :: Django :: 4.0",
         "Framework :: Django :: 4.1",
```
Comment (smithdc1) [inline setup.py:19]:
```suggestion
        "Framework :: Django :: 4.1",
        "Framework :: Django :: 4.2",
        "Framework :: Django :: 5.0",
```

**Pull request merged**